tests/components/august/test_binary_sensor.py

```diff
@@ -69,3 +70,21 @@
         "binary_sensor.k98gidt45gul_name_ding"
     )
     assert binary_sensor_k98gidt45gul_name_ding.state == STATE_OFF
+
+
+async def test_create_doorbell_offline(hass):
+    """Test creation of a doorbell that is offline."""
+    doorbell_one = await _mock_doorbell_from_fixture(hass, "get_doorbell.offline.json")
+    doorbell_details = [doorbell_one]
+    await _create_august_with_devices(hass, doorbell_details)
+
+    binary_sensor_tmt100_name_motion = hass.states.get(
+        "binary_sensor.tmt100_name_motion"
+    )
+    assert binary_sensor_tmt100_name_motion.state == STATE_UNAVAILABLE
+    binary_sensor_tmt100_name_online = hass.states.get(
+        "binary_sensor.tmt100_name_online"
+    )
+    assert binary_sensor_tmt100_name_online.state == STATE_OFF
+    binary_sensor_tmt100_name_ding = hass.states.get("binary_sensor.tmt100_name_ding")
+    assert binary_sensor_tmt100_name_ding.state == STATE_UNAVAILABLE
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5068548)